src/services/dashboard/modules/all-summary/AllSummary.vue

```diff
@@ -27,13 +27,15 @@
                         <span v-if="activeTab === 'billing'" class="suffix">(USD)</span>
                     </div>
                     <div class="toggle-button-group">
-                        <p-button v-for="(d, idx) in dateTypes"
-                                  :key="idx"
-                                  :class="{'selected': selectedDateType === d.name}"
-                                  @click="handleChangeDateType(d.name)"
+                        <p-select-button v-for="(d, idx) in dateTypes"
+                                         :key="idx"
```

Comment:
key를 index로만 하면 다른 vue element들과 겹칠 우려가 있어서, 조금만 특이했으면 좋겠어요! 이렇게?
```suggestion
                                         :key="`date-${d.name}-${idx}`"
```

Reply:
다른 곳들두요!